src/components/code-line.component.js

```diff
@@ -2,66 +2,81 @@
 /* eslint-disable no-nested-ternary */
 
 import React, { Component } from 'react';
-import { View, Text, StyleSheet, Dimensions } from 'react-native';
+import { Text, Dimensions } from 'react-native';
 import SyntaxHighlighter from 'react-native-syntax-highlighter';
 import { getLanguage } from 'lowlight';
 import { github as GithubStyle } from 'react-syntax-highlighter/dist/styles';
-import { colors, fonts, normalize } from 'config';
-
-const styles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-  },
-  wrapper: {
-    flexDirection: 'row',
-    flex: 1,
-  },
-  codeLineContainer: {
-    minWidth: Dimensions.get('window').width - 80,
-    flex: 0.85,
-  },
-  codeLine: {
-    ...fonts.fontCode,
-    fontSize: normalize(11),
-    paddingHorizontal: 10,
-    paddingVertical: 3,
-  },
-  newChunkLineNumbers: {
-    backgroundColor: colors.codeChunkLineNumberBlue,
-  },
-  newChunkLineContainer: {
-    backgroundColor: colors.codeChunkBlue,
-  },
-  newChunkLine: {
-    color: colors.grey,
-  },
-  addLine: {
-    backgroundColor: colors.addCodeGreen,
-  },
-  addLineNumbers: {
-    backgroundColor: colors.addCodeLineNumberGreen,
-  },
-  delLine: {
-    backgroundColor: colors.delCodeRed,
-  },
-  delLineNumbers: {
-    backgroundColor: colors.delCodeLineNumberRed,
-  },
-  lineNumbers: {
-    width: 80,
-    paddingLeft: 10,
-    paddingVertical: 3,
-    flexDirection: 'row',
-    justifyContent: 'space-between',
-  },
-  codeLineNumber: {
-    ...fonts.fontCode,
-    fontSize: normalize(11),
-    flex: 1,
-    alignItems: 'center',
-    color: colors.grey,
-  },
-});
+import { colors, normalize, styledFonts } from 'config';
+import styled, { css } from 'styled-components/native';
+
+function addOrDelLineColors(change) {
+  const lineColors = {
+    lineNumbers: '',
+    line: '',
+  };
+
+  if (change.type === 'add') {
+    lineColors.lineNumbers = colors.addCodeLineNumberGreen;
+    lineColors.line = colors.addCodeGreen;
+  } else if (change.type === 'del') {
+    lineColors.lineNumbers = colors.delCodeLineNumberRed;
+    lineColors.line = colors.delCodeRed;
+  }
+
+  return lineColors;
+}
+
+const Container = styled.View`
+  flex-direction: row;
+`;
+
+const Wrapper = styled.View`
+  flex-direction: row;
+  flex: 1;
+`;
+
+const CodeLineContainer = styled.View`
+  min-width: ${Dimensions.get('window').width - 80}
+  flex: 0.85;
+  background-color: ${props =>
+    props.newChunk
+      ? colors.codeChunkBlue
+      : addOrDelLineColors(props.change).line}
+  }
+`;
+
+const CodeLineStyled = styled.Text`
+  font-family: ${styledFonts.fontCode}
+  font-size: ${normalize(11)}
+  padding: 3 10;
```

Comment:
@josenaranjo Have you ever tested about this kind of shorthands properties? As far as I know, it will cause a crash. [[Link](https://www.styled-components.com/docs/basics#react-native)]